### PR TITLE
[ImageEditing][TCSACR-490] Add new guide for dotnet image editing

### PR DESCRIPTION
**Pull request by hsgwon**
### Change Description ###

Add new guide for dotnet image editing

### API Changes ###

In case of ACR, put the link of ACR

For instance,

 - ACR: TCSACR-490

### CODE REVIEW

File: docs/application/dotnet/guides/multimedia/image-edit.md [hunk 1]
```diff
@@ -0,0 +1,343 @@
...
+   var imageTransformer = new ImageTransformer();
+   var packet = MediaPacket.Create(new VideoMediaFormat(MediaFormatVideoMimeType.I420, 640, 480));
+   ```
+
+<a name="image_colorspace"></a>
+## Convert image color space
+
+To convert one color space of an image to another, follow these steps:
+
+1. Create a `ColorSpaceTransform` instance:
```
Comment (safir-srbd) [inline docs/application/dotnet/guides/multimedia/image-edit.md:65]: Link ColorspaceTransform (https://docs1.stg.tizen.org/application/dotnet/api/TizenFX/latest/api/Tizen.Multimedia.Util.ColorSpaceTransform.html)

File: docs/application/dotnet/guides/multimedia/image-edit.md [hunk 2]
```diff
@@ -0,0 +1,343 @@
...
+   var colorSpaceTransform = new ColorSpaceTransform(ColorSpace.I420);
+   ```
+
+   or, the color space of target image can be set by `ColorSpace` property after `ColorSpaceTransform` is created.
+
+   ```csharp
+   colorSpaceTransform.ColorSpace = ColorSpace.NV12;
+   ```
+
+2. Execute the transformation using `TransformAsync()` of `ImageTransformer` class:
```
Comment (safir-srbd) [inline docs/application/dotnet/guides/multimedia/image-edit.md:77]: Link ImageTransformer (https://docs1.stg.tizen.org/application/dotnet/api/TizenFX/latest/api/Tizen.Multimedia.Util.ImageTransformer.html)

File: docs/application/dotnet/guides/multimedia/image-edit.md [hunk 3]
```diff
@@ -0,0 +1,343 @@
...
+
+   > [!NOTE]
+   > `TransformAsync` using `ColorSpaceTransform` only converts the color space. These functions do not change the image width, height, or any other image property. Due to these restrictions of the image processing library, not all color space combinations are supported for conversion.
+
+<a name="resize_image"></a>
+## Resize image
+
+To resize an image, follow these steps:
+
+1. Create a `ResizeTransform` instance:
```
Comment (safir-srbd) [inline docs/application/dotnet/guides/multimedia/image-edit.md:91]: Link ResizeTransform (https://docs1.stg.tizen.org/application/dotnet/api/TizenFX/latest/api/Tizen.Multimedia.Util.ResizeTransform.html)

File: docs/application/dotnet/guides/multimedia/image-edit.md [hunk 4]
```diff
@@ -0,0 +1,343 @@
...
+   var resizeTransform = new ResizeTransform(new Size(320, 240));
+   ```
+
+   or, the size of target image can be set by `Size` property after `ResizeTransform` is created.
+
+   ```csharp
+   resizeTransform.Size = new Size(1280, 730);
+   ```
+
+2. Execute the transformation using `TransformAsync()` of `ImageTransformer` class:
```
Comment (safir-srbd) [inline docs/application/dotnet/guides/multimedia/image-edit.md:103]: Link ImageTransformer (https://docs1.stg.tizen.org/application/dotnet/api/TizenFX/latest/api/Tizen.Multimedia.Util.ImageTransformer.html)

File: docs/application/dotnet/guides/multimedia/image-edit.md [hunk 5]
```diff
@@ -0,0 +1,343 @@
...
+
+   > [!NOTE]
+   > The image format has no effect on the transformation. If the color space is YUV, then the width and height of the target image must be multiples of eight. This restriction does not apply to the RGB images.
+
+<a name="rotate_image"></a>
+## Rotate image
+
+To rotate an image, follow these steps:
+
+1. Create a `RotateTransform` instance:
```
Comment (safir-srbd) [inline docs/application/dotnet/guides/multimedia/image-edit.md:117]: Link RotateTransform (https://docs1.stg.tizen.org/application/dotnet/api/TizenFX/latest/api/Tizen.Multimedia.Util.RotateTransform.html)

File: docs/application/dotnet/guides/multimedia/image-edit.md [hunk 6]
```diff
@@ -0,0 +1,343 @@
...
+   var rotateTransform = new RotateTransform(Rotation.Rotate90);
+   ```
+
+   or, the rotation of target image can be set by `Rotation` property after `RotateTransform` is created.
+
+   ```csharp
+   rotateTransform.Rotation = Rotation.Rotate180;
+   ```
+
+2. Execute the transformation using `TransformAsync()` of `ImageTransformer` class:
```
Comment (safir-srbd) [inline docs/application/dotnet/guides/multimedia/image-edit.md:129]: Link ImageTransformer (https://docs1.stg.tizen.org/application/dotnet/api/TizenFX/latest/api/Tizen.Multimedia.Util.ImageTransformer.html)

File: docs/application/dotnet/guides/multimedia/image-edit.md [hunk 7]
```diff
@@ -0,0 +1,343 @@
...
+
+   > [!NOTE]
+   > The image format has no effect on the transformation. If the color space is YUV, then the width and height of the target image must be multiples of eight. This restriction does not apply to the RGB images.
+
+<a name="crop_image"></a>
+## Crop image
+
+To crop an image, follow these steps:
+
+1. Create a `CropTransform` instance:
```
Comment (safir-srbd) [inline docs/application/dotnet/guides/multimedia/image-edit.md:143]: Link CropTransform (Link ImageTransformer (https://docs1.stg.tizen.org/application/dotnet/api/TizenFX/latest/api/Tizen.Multimedia.Util.ImageTransformer.html))

File: docs/application/dotnet/guides/multimedia/image-edit.md [hunk 8]
```diff
@@ -0,0 +1,343 @@
...
+   ```
+
+   or, the area to be cropped can be set by `Location` or `Size` property after `CropTransform` is created.
+
+   ```csharp
+   cropTransform.Location = new Point(10, 10);
+   cropTransform.Size = new Size(200, 200);
+   ```
+
+2. Execute the transformation using `TransformAsync()` of `ImageTransformer` class:
```
Comment (safir-srbd) [inline docs/application/dotnet/guides/multimedia/image-edit.md:156]: Link ImageTransformer (https://docs1.stg.tizen.org/application/dotnet/api/TizenFX/latest/api/Tizen.Multimedia.Util.ImageTransformer.html)

**Review by safir-srbd — COMMENTED** [review]
@hsgwon Hello. Hope you are doing well. I have gone through the file, and have suggested some changes for you to implement. Please implement them at your earliest convenience. Thanks!

File: docs/application/dotnet/guides/multimedia/image-edit.md (hunk 2 above)
Comment (hsgwon) [reply docs/application/dotnet/guides/multimedia/image-edit.md:77]: I'll add the link for ImageTransformaer at line 54.

**Review by safir-srbd — APPROVED** [review]
@hsgwon Thank you for making the changes. Everything seems to be in order now.

@asm-foysal @mijin85cho I am approving this PR for merging.

**Pull request merged**